website/docs/concepts/reading.mdx

```diff
@@ -60,7 +61,7 @@ solutions to obtain one from widgets.
 
 <ConditionalSnippet>
 
-#### Extending ConsumerWidget instead of StatelessWidget
+#### 1. Extending ConsumerWidget instead of StatelessWidget
```

Comment:
Could you remove those numbers? That's not a practice the rest of the website uses.